src/gui/resources/translations/de.ts

```diff
@@ -0,0 +1,2749 @@
+        <source>Remove</source>
+        <translation>Entfernen</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.ui" line="1362"/>
+        <source>Set delay to default values: 20us up to 15 chars, 200us above</source>
+        <translation>Verzögerung auf Standardwerte setzen: 20us bis zu 15 Zeichen, 200us darüber</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.ui" line="1378"/>
+        <source>Delay will be the same as it was recorded</source>
+        <translatorcomment>Verzögerung wie bei Aufzeichnung</translatorcomment>
+        <translation>Die Verzögerung ist dieselbe wie bei der Aufnahme</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.ui" line="1365"/>
+        <source>de&amp;fault</source>
+        <translation>&amp;Standard</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.ui" line="1381"/>
+        <source>as t&amp;yped</source>
+        <translation>&amp;Wie eingegeben</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.ui" line="1620"/>
+        <source>Clear</source>
+        <translation>Leeren</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.ui" line="1650"/>
+        <source>Unbind</source>
+        <translation>Zuordnung aufheben</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.ui" line="1657"/>
+        <source>Reset to Default</source>
+        <translation>Auf Standard zurücksetzen</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.ui" line="1664"/>
+        <source>Cancel</source>
+        <translation>Abbrechen</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.ui" line="1671"/>
+        <source>Apply</source>
+        <translation>Anwenden</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.cpp" line="458"/>
+        <source>Key %1 (%2) is pressed but never released.
+This will result in the key being pressed by the macro until you manually press the key itself and release it.
+
+Are you sure you want to continue?</source>
+        <translation>Taste %1 (%2) wird gedrückt, aber nicht losgelassen.
+Dies führt dazu, dass die Taste vom Makro gedrückt wird, bis Sie die Taste selbst manuell drücken und loslassen.
+
+Sind Sie sicher, dass Sie fortfahren möchten?</translation>
+    </message>
+    <message>
+        <location filename="../../rebindwidget.cpp" line="461"/>
+        <source>Key %1 (%2) is released but never pressed.
+This will have no observable effect unless the key is held down manually or by another macro.
+
+Are you sure you want to continue?</source>
+        <translation>Die Taste %1 (%2) wurde losgelassen, aber nie gedrückt.
```

Comment:
"wurde [...] nie" -> "wird [...] nicht" to be consistent with the pressed-but-never-released text?